Lido-Script/script.js

```diff
@@ -88,13 +96,23 @@ const createObolCluster = async (clusterConfig) => {
 // Create all clusters. Should consider graceful retrying here if the number of clusters is too large.
 async function createMultipleClusters() {
   try {
+    const clusters = await constructClustersFromCSV();
+
     const promises = clusters.map(async (clusterConfig) => {
       return await createObolCluster(clusterConfig);
     });
 
     const results = await Promise.all(promises);
 
     console.log(results, "results");
+    
+    writeStream.end();
```

Comment:
Any particular reason why you start the file opening process up on line 10-14 in a script style and then handle the closing here in a function? Would it make sense to do both within this try/catch block?

Reply:
Cause ill need to pass it to the createObolCluster in that case, except if we want to write to csv after we get results line 106, instead of inside of createObolCluster?